### PR TITLE
Fixed hidden tooltips when always on top is active

### LINKED ISSUE #2039: Previews/popup info appearing behind app when "Always on top" enabled
**Describe the bug**
When the "Always on top" option is enabled, previews for emotes/links appear behind the app 

**To reproduce**
Enable "Always on top" and hover mouse on top of an emote or link (nothing will seem to appear because it's behind the app)

**Screenshots**
Screenshot 2020-10-06 at 13 12 10

**Chatterino version**
Chatterino 2.2.2 (commit 07fc0c3ad)

**Operating system**
macOS Big Sur 11.0

### PR DESCRIPTION
**Pull request by revolter**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Fixes #2039.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Bugfix: Fixed hidden tooltips when always on top is active (#2384)
```
Comment (zneix) [inline CHANGELOG.md:5]: This changelog entry should be lower, next to other entries prefixed with `Bugfix`

Comment (revolter) [reply CHANGELOG.md]: Done

**Comment by revolter** [comment]
@pajlada, Could you please take a look at this one too?

**Review by pajlada — APPROVED** [review]
Changed nothing for me on Arch Linux using i3, doesn't seem to be an issue in the first place on Windows either so if this fixes it for macOS then I'm happy
:+1: Thanks for the contribution

Feel free to open a separate PR adding yourself to the contributors list https://github.com/Chatterino/chatterino2/blob/master/resources/contributors.txt#L1-L2

**Pull request merged**